app/components/ActionLinks/tests/index.test.js

```diff
@@ -83,3 +58,22 @@ it('should show PECS link when the user has a PECS role', () => {
 
   expect(wrapper.find('ActionLink').prop('url')).toBe('http://pecs/')
 })
+
+it('should show global search link when the user has isGlobalSearch', () => {
+  const wrapper = mount(<ActionLinks prisonStaffHubUrl="http://" isGlobalSearch />)
+  expect(wrapper.text()).toMatch('Global search')
+})
+
+it('should show roll check link when the user has isEstablishmentRollCheck', () => {
+  const wrapper = mount(<ActionLinks prisonStaffHubUrl="http://" isEstablishmentRollCheck />)
+  expect(wrapper.text()).toMatch('Establishment roll check')
```

Comment:
it was probably nicer before - checking the url was created correctly..

Reply:
the problem I was having with the previous method was the tests were failing due to 2 nodes being returned rather than 1. I understood that to mean, 2 objects were being returned. I think this was because the new link matched 3 other links as they all used prisonStaffHubUrl as a filter.

Reply:
I thought we managed to get round that with the `ActionLink[testId="..."]`